fuzzers/baby/baby_fuzzer/src/main.rs

```diff
@@ -24,14 +24,15 @@ use libafl_bolts::{current_nanos, nonzero, rands::StdRand, tuples::tuple_list, A
 
 /// Coverage map with explicit assignments due to the lack of instrumentation
 static mut SIGNALS: [u8; 16] = [0; 16];
+#[allow(static_mut_refs)] // only a problem in nightly
```

Comment:
In any case these allows have to go

Reply:
Or otherwise leave a biiiiiig TODO comment

Reply:
I've created an issue (#2786) and added a comment to the code with a link to it.